python/lsst/ap/pipe/matchSourceInjected.py

```diff
@@ -366,3 +354,84 @@ def _trimFakeCat(self, fakeCat, image):
         isContainedXy &= ys + self.config.trimBuffer <= bbox.maxY
 
         return fakeCat[isContainedRaDec & isContainedXy]
+
+
+class MatchInjectedToAssocDiaSourceConnections(
+    PipelineTaskConnections,
+    defaultTemplates={"coaddName": "deep",
+                      "fakesType": "fakes_",
+                      "injection_prefix": "injection_",
+                      "injected_prefix": "injected_"},
```

Comment:
I see `injection_prefix` and `injected_prefix` in other connections classes, but don't see them actually used. Are they necessary?